### PR TITLE
Added LexiNomicon to list of projects

### PR DESCRIPTION
**Pull request by Flaxscrip**

### CODE REVIEW

File: Projects/README.md [hunk 1]
```diff
@@ -23,3 +23,4 @@ Each of the projects below are all self-funded and community-driven. If one (or
 * [Tesseract](Tesseract.md) - Open Crypto Equity and Assets Network
 * [Meridion](Meridion.md) - Open Project Governance-as-a-Service
 * [Castle in the Cloud](Castle-in-the-Cloud.md) - Easy DIY Sovereign Cloud Server
+* [LexiNomicon](Lexinomicon.md) - Nomicon gamespace for governance of Ulex bounties
```
Comment (macterra) [inline Projects/README.md]: Where is Lexinomicon.md? We should be consistent with the spelling, maybe Lexinomicon (following the convention set by Cryptonomicon), not LexiNomicon. Or should we change to CryptoNomicon?

Comment (Flaxscrip) [reply Projects/README.md]: Just added the Lexinomicon.md to the pull request.

File: Projects/Lexinomicon.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+The [Lexinomicon](https://github.com/cryptotechguru/Lexinomicon) is a [Nomicon game](https://github.com/cryptotechguru/Cryptonomicon/tree/master/Nomicon) providing governance services and a collaboration space for the [Ulex](https://github.com/proftomwbell/Ulex/blob/master/versions/1.1/ulex-europaeus.png) bounty system.
```
Comment (macterra) [inline Projects/Lexinomicon.md]: Ulex should link to the repo, not the image.

File: Projects/README.md (hunk 1 above)
Comment (macterra) [reply Projects/README.md]: Still says LexiNomicon in the link.

Comment (Flaxscrip) [reply Projects/README.md]: Fixed.

**Pull request merged**